### PR TITLE
CRUD for clients

### PR DESCRIPTION
**Pull request by AT0047**
1.     a database table clients made to store the data
2.     an index view page to display the data
3.     a create view with form to store the data
4.     an edit view with form to edit the data
5.     route resource with /clients in web route file
6.     a controller for client
7.     a request file to validate the fore data

### CODE REVIEW

File: resources/views/clients/create.blade.php [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+                <h4 class="mt-3 mb-3">Create a client</h4>
+            </div><!--t-manage-->
+        </div><!--container-->
+        <form class="d-flex" action="{{ route('clients.store') }}" method="POST">
+            @csrf
+            <div class="create-c-i my-2">
+                <div class="row">
+                    <div class="col-md-4  my-2">
+                        <div class="col-5 ">
+                            <label for="inputname0">Company / Business name * : </label>
```
Comment (MuhammadMamadouh) [inline resources/views/clients/create.blade.php:18]: lable names must be translated 
ex: __(label_name)

File: resources/views/clients/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+                                    value='0'>No</label>
+                            </div>
+                        </div>
+                    </div><!--18-->
+                    <div class="col-md-4 my-2">
+                        <div class="col-5 ">
+                            <label for="inputname19"> Collector :</label>
+                        </div>
+                        <div class="col-7 ">
+                            <select name="collector_id" class="form-select" id="inputname19">
```
Comment (MuhammadMamadouh) [inline resources/views/clients/create.blade.php:231]: collectors are dynamic values not static ones
get them from users who has role collector

File: resources/views/clients/create.blade.php [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+                            <div class="form-check">
+                                <input class="form-check-input" type="checkbox" id="exampleCheckbox2">
+                                <label class="form-check-label" for="exampleCheckbox2" name="recovery_cost"
+                                    value='0'>No</label>
+                            </div>
+                        </div>
+                    </div><!--21-->
+                    <div class="col-md-4 my-2">
+                        <div class="col-5 ">
+                            <label for="inputname22"> Collection scenario :</label>
```
Comment (MuhammadMamadouh) [inline resources/views/clients/create.blade.php:273]: collection scenario also as collectors

File: resources/views/clients/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,592 @@
...
+                            <option value="3">Relance team back office</option>
+                            <option value="3">Retail</option>
+                            <option value="3">Salomon</option>
+                            <option value="3">Test 2</option>
+                            <option value="3">US Clients</option>
+                        </select>
+                    </div>
+                    <div class="form-group col-md-4 col-sm-6">
+                        <select name="onlyScenario" class="m-2 form-select" data-placeholder="Collection scenario:"
+                            title="Collection scenario">
```
Comment (MuhammadMamadouh) [inline resources/views/clients/index.blade.php:63]: collection scenario also should be got from collection scenario table

File: app/Http/Controllers/ClientController.php [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+use Illuminate\Support\Facades\Log;
+
+class ClientController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        $clients = Client::all();
```
Comment (MuhammadMamadouh) [inline app/Http/Controllers/ClientController.php:18]: use pagination with number 30

File: app/Http/Controllers/ClientController.php [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        return view('clients.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(ClientRequest $request)
+    {
+
+            Client::create($request->all());
```
Comment (MuhammadMamadouh) [inline app/Http/Controllers/ClientController.php:36]: use request validated not request all

File: resources/views/clients/edit.blade.php
Comment (MuhammadMamadouh) [inline resources/views/clients/edit.blade.php:1]: update edit after changing creates with requested changes

**Review by MuhammadMamadouh — CHANGES_REQUESTED** [review]

File: resources/views/clients/create.blade.php (hunk 1 above)
Comment (AT0047) [reply resources/views/clients/create.blade.php:18]: done

File: resources/views/clients/create.blade.php (hunk 2 above)
Comment (AT0047) [reply resources/views/clients/create.blade.php:231]: waiting for the table

File: resources/views/clients/create.blade.php (hunk 3 above)
Comment (AT0047) [reply resources/views/clients/create.blade.php:273]: waiting for the table

File: resources/views/clients/index.blade.php (hunk 4 above)
Comment (AT0047) [reply resources/views/clients/index.blade.php:63]: waiting for the table

File: app/Http/Controllers/ClientController.php (hunk 6 above)
Comment (AT0047) [reply app/Http/Controllers/ClientController.php:36]: done

File: app/Http/Controllers/ClientController.php (hunk 5 above)
Comment (AT0047) [reply app/Http/Controllers/ClientController.php:18]: done

**Pull request closed without merging**